<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductDialog</class>
 <widget class="QDialog" name="ProductDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add / Edit Product</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelQuantity">
       <property name="text">
        <string>Quantity:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelUnit">
       <property name="text">
        <string>Unit:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelDescription">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leName"/>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="sbQuantity"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="leDescription"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cbUnit">
       <item>
        <property name="text">
         <string>pcs</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mÂ²</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="ProductButtonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Save</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ProductButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProductDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ProductButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProductDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
